Make getting started page more friendly (#2045)
fluent/fluent-bit-docs: installation/getting-started-with-fluent-bit.md

<img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=e9732f9c-44a4-46d3-ab87-86138455c698" />

> The Fluent Bit telemetry agent is available for Linux, macOS, Windows, and BSD family operating systems.

Reviewer: Should "BSD family" be hyphenated? I think so, as it's a compound word. BSD-family.

Author: let's be honest, I'd like to remove that part entirely, but this is a direct yank from the top page.

> If you are interested in learning about Fluent Bit you can try out the [sandbox environment](../about/sandbox-and-lab-resources.md).

Reviewer: ```suggestion
If you're interested in learning about Fluent Bit, you can try out the [sandbox environment](../about/sandbox-and-lab-resources.md).
```

## Requirements

All Fluent Bit installations must meet certain [requirements](./requirements.md).

## Supported platforms and packages

> Fluent Bit software is available for a variety of platforms and can be installed
> using several methods.

Reviewer: ```suggestion
Fluent Bit software is available for a variety of platforms and can be installed using several methods.
```

You can [direct download and compile the source code](../installation/sources/README.md).

You can also select a package follow the specific instructions for it using the following links:

### Container deployment

| Deployment Type   | Instructions |
| ----------------- | -------------------------------------------------- |
| Kubernetes        | [Deploy on Kubernetes](kubernetes.md#installation) |
| Docker            | [Deploy with Docker](docker.md)                    |

> | Containers on AWS | [Deploy on Containers on AWS](aws-container.md)    |

Author: I'm planning on moving all these tables as it is.

### Install on Windows (packages)

| Operating System    | Installation instructions |
| ------------------- | ------------------------- |
| Windows Server 2019 | [Windows Server `EXE`](windows.md#installing-from-exe-installer), [Windows Server ZIP](windows.md#installing-from-zip-archive) |
| Windows 10 2019.03  | [Windows `EXE`](windows.md#installing-from-exe-installer), [Windows ZIP](windows.md#installing-from-zip-archive)               |

### Install on macOS (packages)

| Operating System    | Installation instructions                     |
| ------------------- | --------------------------------------------- |
| macOS               | [Homebrew](macos.md#installing-from-homebrew) |

### Compile from source (Linux, Windows, FreeBSD, macOS)

| Operating system | Installation instructions                                   |
| ---------------- | ----------------------------------------------------------- |
| Linux, FreeBSD   | [Compile from source](sources/build-and-install.md)         |
| macOS            | [Compile from source](macos.md#compile-from-source) |
| Windows          | [Compile from Source](windows.md#compile-from-source)       |

### Enterprise packages

> Fluent Bit packages are also provided by [enterprise providers](https://fluentbit.io/enterprise) for older end of life versions, Unix systems, or for additional support and features including aspects like CVE backporting.

Reviewer: ```suggestion
Fluent Bit packages are also provided by [enterprise providers](https://fluentbit.io/enterprise) for older end-of-life versions, Unix systems, or for additional support and features including aspects (such as CVE backporting).
```
